src/components/views/settings/ChangePassword.js

```diff
@@ -53,7 +53,7 @@ module.exports = React.createClass({
             onCheckPassword: function(oldPass, newPass, confirmPass) {
                 if (newPass !== confirmPass) {
                     return {
-                        error: _t("New passwords don't match") + "."
```

Comment:
We consistently don't put a `.` on other errors in this component, so it's easier to remove this one.